test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/BuildServerTestFixture.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using Microsoft.AspNetCore.Razor.Tools;
+
+namespace Microsoft.AspNetCore.Razor.Design.IntegrationTests
+{
+    public class BuildServerTestFixture : IDisposable
+    {
+        private static readonly TimeSpan _defaultShutdownTimeout = TimeSpan.FromSeconds(60);
+
+        public BuildServerTestFixture()
+        {
+            PipeName = Guid.NewGuid().ToString();
```

Comment:
Would there be benefit in having the name be deterministic name? In the event you want to query the server to debug?

Reply:
I'm not sure I see the advantage in that. This is only generated once per fixture. And we use the same pipe name in all our tests. Could you give me an example for when that would be useful?